src/registrar/migrations/0032_alter_transitiondomain_status.py

```diff
@@ -1,5 +1,3 @@
-# Generated by Django 4.2.1 on 2023-09-27 00:18
```

Comment:
**suggestion (non-blocking)**: It looks like this PR doesn't make any change to this migration file. Ideally, to avoid distraction this change wouldn't be present in this PR.